### ISSUE TITLE
The NVDA is not announcing "Create Function App in Azure" header text of "Create Function App in Azure" dialog.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode#138120]
### Actual:
When we activate the "Create Function App in Azure" control the NVDA only announces "edit Enter a globally unique name for the new function app. (Press 'Enter' to confirm or 'Escape' to cancel) blank" but it does not announce the header text that is "Create Function App in Azure (1/3)".

**This issue is also observed with JAWS.**
This issue is also observed with scenarios such as "Create new project" and "Create Function App in Azure(Advanced) (1/7)" dialog

### Expected:
After activating the "Create Function App in Azure" control the NVDA should announce "Create Function App in Azure(1/3)" header text.

**GitHubTags:**#A11yMAS;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win10;#NVDA;#JAWS;#Element:Header;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1
Screen Reader: NVDA(2021.1),JAWS(2021.2107.12 ILM)

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open. TAB to "Install" button and hit ENTER key to install it.
4. "Azure" button will appear in navigation pane. TAB to it and select it.
5. Start NVDA.
6. TAB to any Subscription and expand options using Shift+F10 then TAB to "Create Function App in Azure" control and select it.
7. Observe the announcement by screen reader after activating the "Create Function App in Azure" control.

### User Impact:
The screen reader users won't be able to understand which popup or dialog they are navigating.

“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

**Comment by jinglouMSFT** [comment]
This is similar to the Git:Clone comment where the content of the control was not read by NVDA.
